src/reaper_osara.cpp

```diff
@@ -277,6 +277,37 @@ void resetTimeCache(TimeFormat excludeFormat) {
 	}
 }
 
+string FormatNoteLength(double start, double end) {
+	int measureLength;
+	double startBeats;
+	double endBeats;
+	TimeMap2_timeToBeats(NULL, start, NULL, &measureLength, &startBeats, NULL);
```

Comment:
nit: NULL -> nullptr
```suggestion
	TimeMap2_timeToBeats(nullptr, start, nullptr, &measureLength, &startBeats, nullptr);
```